src/panels/lovelace/components/hui-card-edit-mode.ts

```diff
@@ -185,21 +232,21 @@ export class HuiCardEditMode extends LitElement {
     this._editCard();
   }
 
-  private _handleAction(ev: CustomEvent<ActionDetail>) {
-    switch (ev.detail.index) {
-      case 0:
+  private _handleAction(ev) {
+    switch ((ev.target as any).action) {
```

Comment:
```suggestion
    switch (ev.target.action) {
```

`ev` is `any` either way.